packages/bitsight/data_stream/vulnerability/manifest.yml

```
@@ -0,0 +1,67 @@
+---
+title: "Bitsight Vulnerability"
+type: logs
+streams:
+  - input: cel
+    title: Bitsight Vulnerability
+    description: Collect vulnerability data from Bitsight Threats API
+    template_path: cel.yml.hbs
+    vars:
+      - name: batch_size
+        type: integer
+        title: Batch Size
+        description: Number of records retrieved per request.
+        required: true
+        show_user: true
+        default: 100
+      - name: interval
+        type: text
+        title: Poll Interval
+        description: Defines how often data is collected from the Bitsight API.
+        required: true
+        show_user: true
+        default: 5m
+      - name: initial_interval
+        type: text
+        title: Initial Lookback Interval
+        description: Defines how far back the first query should go when no cursor is set. Allowed time units are "s", "m", "h".
+        required: true
+        show_user: true
+        default: 720h
+      - name: tags
+        type: text
+        title: Tags
+        description: Tags to apply to each event.
+        multi: true
+        required: false
+        show_user: false
+        default:
+          - forwarded
+          - bitsight-vulnerability
+      - name: preserve_original_event
+        type: bool
+        title: Preserve original event
+        description: Preserves a raw copy of the evidence payload in `event.original`.
+        required: false
+        show_user: true
+        default: false
+      - name: enable_request_tracer
+        type: bool
+        title: Enable request tracing
+        description: Logs HTTP requests/responses locally for troubleshooting. Disable after collecting evidence.
```

```suggestion
        description: The request tracer logs requests and responses to the agent's local file-system for debugging configurations. Enabling this request tracing compromises security and should only be used for debugging. See [documentation](https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-input-cel.html#_resource_tracer_enable) for details.

```
The link to the documentation is important because of the risks that it introduces in use.